__tests__/Port.test.js

```diff
@@ -0,0 +1,35 @@
+/* globals describe it expect */ 
+const Port = require('../src/Port.js');
+const Ship = require('../src/Ship.js');
+
+describe('Port', () => {
+    describe('with ports and ships', () => {
+        let port;
+
+        beforeEach(() => {
+            port = new Port('Dover');
+        });
+        it('can be instantiated ', () => {
```

Comment:
As before, its a good idea to leave a blank line between each test and describe block just for readability